nb-api/src/main/java/io/nosqlbench/api/metadata/SystemId.java

```diff
@@ -125,4 +172,75 @@ public static String getHostSummary() {
         return gson.toJson(details);
 
     }
+
+    private final static String radixSymbols = "01234567890abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ~-"; // 64 symbols, for 2^6!
```

Comment:
Need to ensure that all characters here are in ordinal order so that alphanumeric sorting still works as expected. More tests needed.